Sources/App/Commands/Ingest.swift

```diff
@@ -24,22 +24,20 @@ struct IngestCommand: AsyncCommand {
     func run(using context: CommandContext, signature: SPICommand.Signature) async throws {
         let client = context.application.client
         let db = context.application.db
-        let logger = Logger(component: "ingest")
 
```

Comment:
```suggestion
        let logger = Logger(component: "ingest")
        Current.setLogger(logger)

```

We can't simply remove this logger here - it needs to be set as the `Current.logger` or else all the subsequent calls will lead to a crash (logger is a `static var logger: Logger!`).